### DISCUSSION TITLE
Welcome to SimplyPluralApi Discussions!

### DISCUSSION THREAD

**Discussion by Cellaryllis** [ApparyllisOrg/SimplyPluralApi#132] [Announcements]
## 👋 Welcome!
  Feel free to post discussions to discuss the state of the API, ask questions on specific code, questions about the API and anything else you can think of related to the API!

**Comment by MarianaCannabis** [comment]
Subject: Keeping Simply Plural alive for users .. a request

Hi Apparyllis team,

First, thank you for everything Simply Plural has meant, and a lot of respect for the years of work behind it. I completely understand the decision to step back; maintaining something at this scale is exhausting, and your wellbeing comes first.

I'm reaching out because many users rely on the app every day, and I'd like to help keep it available for them after the servers go offline on July 1st. I'm a developer and I'm willing to take on the hosting and maintenance myself, at no cost or obligation to you.

To make that possible, I have two requests:

1. Would you be open to releasing the SimplyPluralApi (and ideally the client) under an open-source license, e.g. AGPL-3.0? Right now there's no LICENSE file, so self-hosting isn't legally clear, even though the code is public.

2. Could the client be pointed at a custom server URL (a configurable endpoint), so a self-hosted instance can actually be used? If that's not feasible on your side, I'm happy to do that work in a fork.

I fully respect that the Simply Plural name and logo are your registered trademark. If you'd prefer, I'll run any continued or self-hosted version under a completely different name and logo, that's no problem at all.

There's no pressure and no rush on a decision. I just wanted to offer a path that keeps the tool alive for the people who depend on it, while taking the burden off you entirely.

Thank you for considering it, and either way, thank you for building something that mattered to so many.

Best regards,
Naemi und Andre
